infrastructure-setup/scripts/prepare-nexus.sh

```diff
@@ -3,18 +3,31 @@ export PATH=$PATH:/usr/local/bin/
 
 
 BASE_DIR=${OPENDEVSTACK_DIR:-"/ods"}
+NEXUS_ADMIN_USER="admin"
 cwd=${pwd}
 
 if [ "$HOSTNAME" != "openshift" ] ; then
 	echo "This script has to be executed on the openshift VM"
 	exit
 fi
 
+# Source the core configuration to get nexus informations
+. $BASE_DIR/ods-configuration/ods-core.env
+
+# Source local env informations to get nexus admin user
+if [ ! -f "${BASE_DIR}/local.config" ] ; then
+  echo "Local config file not found, will use default values"
+else
+  echo "Source local env default values"
+  . $BASE_DIR/local.env.config
+fi
+
+
 oc login -u system:admin
 oc project cd
 
 cd /ods/ods-core/nexus/ocp-config
-yes 'y' | tailor update -v --force
+# yes 'y' | tailor update -v --force
```

Comment:
Why is it commented now? BTW, there is an `--non-interactive` flag ...

Reply:
That happened yesterday during local testing and wasn't intended. I have corrected this.